llvm/test/tools/llvm-symbolizer/no-use-symbol-table.s

```diff
@@ -0,0 +1,219 @@
+# REQUIRES: x86-registered-target
```

Comment:
This test should test both `--no-use-symbol-table` and `--use-symbol-table`, and should be named appropriately.